### PR TITLE
Removing agent proto and classes from app_dart

### PR DESCRIPTION
**Pull request by yusufm**
Removing references to agent code in various places in app_dart.

Removed agent references from keyhelper.

Removed agent code from taskservice.

 Removing agent code from update task status in app_data
    1) Removed references to agent requests.
    2) Removed task retry logic which no longer applies to LUCI jobs.
    3) Removed inserting task data into bigquery which LUCI does directly.

Test=app_dart/dart test

### CODE REVIEW

**Review by CaseyHillers — APPROVED** [review]
LGTM

File: app_dart/lib/src/request_handlers/update_task_status.dart [hunk 1]
```diff
@@ -49,22 +42,12 @@ class UpdateTaskStatus extends ApiRequestHandler<UpdateTaskStatusResponse> {
...
-
   @override
   Future<UpdateTaskStatusResponse> post() async {
     checkRequiredParameters(<String>[newStatusParam]);
-    if (requestData.containsKey(taskKeyParam)) {
-      checkRequiredParameters(<String>[taskKeyParam]);
-    } else {
-      checkRequiredParameters(<String>[gitBranchParam, gitShaParam, builderNameParam]);
-    }
+    checkRequiredParameters(<String>[gitBranchParam, gitShaParam, builderNameParam]);
```
Comment (CaseyHillers) [inline app_dart/lib/src/request_handlers/update_task_status.dart:50]: nit: concat this with the line above

File: app_dart/lib/src/request_handlers/update_task_status.dart [hunk 2]
```diff
@@ -144,28 +111,9 @@ class UpdateTaskStatus extends ApiRequestHandler<UpdateTaskStatusResponse> {
 
   /// Retrieve [Task] to update from [DatastoreService].
   Future<Task> _getTask(DatastoreService datastore) async {
```
Comment (CaseyHillers) [inline app_dart/lib/src/request_handlers/update_task_status.dart:113]: _getTask should be removed, and directly call _getTaskFromNamedParams

Comment (yusufm) [reply app_dart/lib/src/request_handlers/update_task_status.dart]: done

File: app_dart/lib/src/request_handlers/update_task_status.dart (hunk 1 above)
Comment (yusufm) [reply app_dart/lib/src/request_handlers/update_task_status.dart]: done

**Pull request merged**